pyproject.toml

```diff
@@ -11,7 +11,7 @@ authors = [
 # PEP 621 project metadata
 # See https://www.python.org/dev/peps/pep-0621/
 dependencies = [
-    "numpy>=1.24.2",
+    "numpy<2.0.0,>=1.24.2",
```

Comment:
Do not cap to 2.0

Reply:
Remove <2.0.0 version cap in https://github.com/leifdenby/neural-lam/commit/f0a151b6a4d78738f2c50d28a83533521fa29931